production.yml

```diff
@@ -35,7 +35,7 @@ services:
     ports:
       - "6667:5000"
     volumes:
-      - static:/app/pra_request_tracker/static
+      - static:/app/pra_request_tracker/pra_request_tracker/static
```

Comment:
Oh this was something I found out yesterday, since this the app is running `collectstatic` as part of `/entrypoint`, we actually want this to point to `/app/staticfiles` instead

Reply:
I also had to add this to the dockerfile too:
```
RUN mkdir -p /app/staticfiles && chown django /app/staticfiles
```